Pod/Classes/Categories/PINImage+WebP.m

```diff
@@ -77,6 +78,62 @@ + (PINImage *)pin_imageWithWebPData:(NSData *)webPData
     return nil;
 }
 
++ (NSData *)pin_DataFromWebPimage:(PINImage *)image {
+    
```

Comment:
Style nit: mind removing the newlines at the beginning and end of this method and putting the brace on a new line?